app/models/components/course/groups_ability_component.rb

```diff
@@ -22,6 +22,7 @@ def allow_group_manager_manage_group
   end
 
   def course_group_manager_hash
-    { group_users: { user_id: user.id, role: Course::GroupUser.roles[:manager] } }
+    { group_users: { course_user_id: user.course_user_ids,
```

Comment:
Sorry. It should be singular, because it's a has_one.

```ruby
  { group_users: { course_user: { user_id: user.id },
                   role: Course::GroupUser.roles[:manager] } }
```